app/task_routes.py

```diff
@@ -0,0 +1,131 @@
+from app import db
+from app.models.task import Task
+from flask import Blueprint, jsonify, abort, make_response, request
+import datetime
+import requests
+from app import token 
```

Comment:
Since `token` is only used in task_routes, I prefer that the variable get created here instead of in app. 

You can still `import os` and get the value of your api key, but you would just do that in your `send_slack_message()` method.

Also in the future you'll work on APIs that make several other external API calls so we should be more specific with the token variable name. Maybe something like `SLACK_API_TOKEN`